cypress/support/step-definitions/configurable-items-steps.js

```diff
@@ -1,20 +1,30 @@
 import {
   draggableItemByText,
   draggableItemByTextNoIframe,
-  draggableItemByPosition
-} from '../../locators/configurable-items';
-import { dragAndDrop } from '../helper';
+  draggableItemByPosition,
+} from "../../locators/configurable-items";
+import { dragAndDrop } from "../helper";
 
 const startPosition = 16;
 
-When('I drag Configurable Items {string} to {int}', (record, destinationId) => {
+When("I drag Configurable Items {string} to {int}", (record, destinationId) => {
   dragAndDrop(draggableItemByText(record), destinationId, startPosition);
 });
 
-When('I drag Configurable Items with iFrame {string} to {int}', (record, destinationId) => {
-  dragAndDrop(draggableItemByTextNoIframe(record), destinationId, startPosition);
-});
+When(
+  "I drag Configurable Items with iFrame {string} to {int}",
```

Comment:
it looks not very good IMO, I would prefer to put all the line back and stick with the `When` keyword

Reply:
This is how prettier formats it, prettier is opinionated so it doesn't matter what we individually prefer